ee/ucp/kubernetes/layer-7-routing.md

```diff
@@ -21,161 +19,12 @@ Use an ingress controller when you want to:
-  name: infra
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: nginx-ingress-service-account
-  namespace: infra
-```
-
-Then apply the namespace and service account via `kubectl`:
-
-```
-🐳  → kubectl apply -f ns-and-sa.yaml
-namespace "infra" created
-serviceaccount "nginx-ingress-service-account" created
-
-```
-
-### Step 3: Create an RBAC Policy
-
-Apply an RBAC role-binding for NGINX controller access to the API Server, such as the one shown in the following example: 
-
-```
-🐳  → cat ingress-rbac.yaml
-## Source: https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/rbac/rbac.yaml
-kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: nginx-ingress-cluster-role
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - services
-  - endpoints
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - secrets
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - get
-  - list
-  - watch
-  - update
-  - create
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - list
-- apiGroups:
-  - ""
-  resources:
-  - events
-  verbs:
-  - create
-  - patch
-- apiGroups:
-  - extensions
-  resources:
-  - ingresses
-  verbs:
-  - list
-  - watch
-  - get
-- apiGroups:
-  - "extensions"
-  resources:
-  - ingresses/status
-  verbs:
-  - update
----
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: nginx-ingress-cluster-rb
-subjects:
-- kind: ServiceAccount
-  name: nginx-ingress-service-account
-  namespace: infra
-roleRef:
-  kind: ClusterRole
-  name: nginx-ingress-cluster-role
-  apiGroup: rbac.authorization.k8s.io
-```
-
-Then apply it with `kubectl`:
-
-```
-🐳  → kubectl apply -f ingress-rbac.yaml
-```
+Kubernetes provides an NGINX ingress controller that is based on the Kubernetes community [NGINX controller](https://github.com/kubernetes/ingress-nginx) and that can be used in Docker EE. This example serves as a recommendation for how to deploy an ingress controller on Docker Enterprise, but it is not directly supported by Docker, Inc.
+Learn about [ingress in Kubernetes](https://v1-11.docs.kubernetes.io/docs/concepts/services-networking/ingress/). 
 
-### Step 4: Deploy NGINX Controller
+## Deploy NGINX ingress controller
 
-Kubernetes provides an NGINX ingress controller that is based on the Kubernetes community [NGINX controller](https://github.com/kubernetes/ingress-nginx), and  can be used in Docker Enterprise Edition, 
+Kubernetes provides an NGINX ingress controller that is based on the Kubernetes community [NGINX controller](https://github.com/kubernetes/ingress-nginx), and can be used in Docker Enterprise Edition, 
 but it is not directly supported by Docker, Inc.
 
 For another example of a YAML NGINX kube ingress deployment, refer to <https://success.docker.com/article/how-to-configure-a-default-tls-certificate-for-the-kubernetes-nginx-ingress-controller>.
```

Comment:
For an example of the.... Its not really another.

Reply:
I think you would end the file here. Lines182 to the end of the file refer to checking a successful installation which I think is covered in the KB.